glustercli/cmd/georep.go

```diff
@@ -346,8 +359,80 @@ func getRemoteClient(host string) (string, *restclient.Client, error) {
 	} else {
 		clienturl = fmt.Sprintf("%s://%s:%d", geoRepHTTPScheme, host, geoRepGlusterdPort)
 	}
-	client, err := restclient.New(clienturl, "", "", "", true)
-	return clienturl, client, err
+
+	secret := ""
```

Comment:
can we rename this variable to `remoteSecret` as this is remote secret?

Reply:
done